### PR TITLE
fix: ProgressRing doesn't collapse on iOS

### PR DESCRIPTION
**Pull request by ghuntley**
GitHub Issue (If applicable): https://github.com/unoplatform/uno/issues/1220

## PR Type

What kind of change does this PR introduce?

- Bugfix
- Project automation

## What is the current behavior?

ProgressRing on iOS when collapsed is visible

image

## What is the new behavior?

Resolved (pending) and added user interface test scenarios to prevent regression

## PR Checklist

Please check if your PR fulfills the following requirements:

- [ ] Tested code with current [supported SDKs](../README.md#supported)
- [ ] Docs have been added/updated which fit [documentation template](https://github.com/nventive/Uno/blob/master/doc/.feature-template.md). (for bug fixes / features)
- [x] [Unit Tests and/or UI Tests](doc/articles/working-with-the-samples-apps.md) for the changes have been added (for bug fixes / features) (if applicable)
- [ ] [Wasm UI Tests](doc/articles/working-with-the-samples-apps.md#running-the-webassembly-ui-tests-snapshots) are not showing unexpected any differences. Validate PR `Screenshots Compare Test Run` results.
- [ ] Contains **NO** breaking changes
- [ ] Updated the [Release Notes](https://github.com/nventive/Uno/tree/master/doc/ReleaseNotes)
- [ ] Associated with an issue (GitHub or internal)

## Other information

Opening up as a draft pull-request early for feedbacl.

### CODE REVIEW

File: src/SamplesApp/UITests.Shared/Windows_UI_Xaml_Controls/Progress/ProgressRing_Active.xaml.cs [hunk 1]
```diff
@@ -14,17 +14,15 @@
...
 using Windows.UI.Xaml.Navigation;
 
-// The Blank Page item template is documented at https://go.microsoft.com/fwlink/?LinkId=234238
-
-namespace SamplesApp.Samples.Progress
+namespace UITests.Shared.Windows_UI_Xaml_Controls.Progress
 {
 	/// <summary>
-	/// An empty page that can be used on its own or navigated to within a Frame.
+	/// ignoreInAutomatedTests set to true because progressrings animate.
```
Comment (ghuntley) [inline src/SamplesApp/UITests.Shared/Windows_UI_Xaml_Controls/Progress/ProgressRing_Active.xaml.cs]: Wondering if folks have any suggestions on how to make progressrings compatible with screenshot testing?

Comment (jeromelaban) [reply src/SamplesApp/UITests.Shared/Windows_UI_Xaml_Controls/Progress/ProgressRing_Active.xaml.cs]: This is actually a test that should be automated. For now iOS still fails to run automatically, but you can script it already for Android/Wasm.

**Comment by carldebilly** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- src/SamplesApp/UITests.Shared/Windows_UI_Xaml_Controls/Progress/ProgressRing_Collapsed_Automated.xaml.cs  1
- src/SamplesApp/UITests.Shared/Windows_UI_Xaml_Controls/Progress/ProgressRing_Active.xaml.cs  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=4826546&bid=16039867)

**Pull request closed without merging**